qutip/stochastic.py

```diff
@@ -183,26 +207,39 @@ class StochasticSolverOptions:
     construct an instance of this class, so it is rarely needed to explicitly
     create an instance of this class.
 
+    Within the attribute list, a ``time_dependent_object`` is either
+
+    - :class:`~Qobj`: a constant term
+    - 2-element list of ``[Qobj, time_dependence]``: a time-dependent term
+      where the ``Qobj`` will be multiplied by the time-dependent scalar.
+
+    For more details on all allowed time-dependent objects, see the
+    documentation for :class:`~QobjEvo`.
```

Comment:
@hodgestar see here for that comment about the `*` section that appeared removed (and the `H`, `c_ops` and `sc_ops` attributes below).

Reply:
Thanks!